tools/ci_build/github/azure-pipelines/stages/nuget-linux-cuda-packaging-stage.yml

```diff
@@ -8,13 +8,11 @@ parameters:
   type: string
 - name: buildJava
   type: boolean
-  default: false
 - name: buildNodejs
   type: boolean
-  default: false
 
 stages:
-- stage: Linux_C_API_Packaging_Combined_CUDA
+- stage: Linux_C_API_Packaging_GPU
```

Comment:
So, right now the stage has two jobs. Your this change split them to two stages. How does it help? Why is this change needed?

Reply:
Why do you need to change this name from "Linux_C_API_Packaging_Combined_CUDA" to "Linux_C_API_Packaging_GPU" ?

Reply:
Oh, now I understand. 
Before your change, this file has one stage which contains 3 jobs.
After your change, this file has two stages. The first one contains two jobs. The second one contains one job. The second stage depends on the first stage. 
Why is it better?

Reply:
The reason doing that because downstream stage like Jar_Packaging_GPU, NuGet_Packaging_GPU and Nodejs_Packaging doesn't depend on the 3rd job, which is now the second stage, this will speed up the whole pipeline allow downstream to run without waiting on the the second stage.